sirepo/status.py

```diff
@@ -52,6 +53,10 @@ def init_apis(*args, **kwargs):
 
 def _run_tests():
     """Runs the SRW "Undulator Radiation" simulation's initialIntensityReport"""
+    pkre(
+        r'"isLoggedIn": true',
```

Comment:
i think it would be better to parse the json to be sure it is completely valid then assert x.isLoggedIn == True (in this case we literally want this, because it should be a boolean.

Reply:
8f3de227